src/utils/apiFunc.jsx

```diff
@@ -1,9 +1,10 @@
 import axios from 'axios'
+import { Axios } from './urlConstants'
 
-export const configFunc = () => {
+export const configFunc = (header) => {
   const userdata = window.localStorage.getItem('userInfo')
   const token = JSON.parse(userdata).token
-  const headers = { 'Content-Type': 'application/json' }
+  const headers = { 'Content-Type': header || 'application/json' }
```

Comment:
in future other header parameters might also be changed so this has been made too specific. it should merge default headers and header passed